src/helpers/fallbackauthdialog.ts

```diff
@@ -8,6 +8,25 @@
 
 import { Configuration, PublicClientApplication, RedirectRequest } from "@azure/msal-browser";
 
+const msalConfig: Configuration = {
+  auth: {
+    clientId: "{application GUID here}", //This is your client ID
```

Comment:
This won't work, right?
It is weird to give a non-working template. Can we somehow get this GUID or make it easier for the client to input it (like in Yo Office)?

I don't think having a template that doesn't work after getting it is good.

Reply:
This was just moved from another place in the file with these already that way.  You're forgetting this is the sso template the requires the extra stop of 'npm run configure-sso' which updates these values with unique and good ones.